File: fctrl2/fctrl2.py
#!/usr/bin/python

def fak(faktorial):
    if faktorial == 1:
        return faktorial;
    return faktorial * fak(faktorial - 1)


if __name__ == "__main__":
    testcases = raw_input("")
    for i in range(0, int(testcases)):
        faktorial = raw_input("")
        print fak(int(faktorial))

File: fctrl2/fctrl2.go
package main

import ( "fmt" )

func fak(faktorial int64) int64 {
	if faktorial == 1 {
		return faktorial
	}
	return faktorial * fak(faktorial - 1)
}

func main() {
	var t int
	fmt.Scanf("%d", &t)
	for i := 0;i < t; i++ {
		var faktor int64
		fmt.Scanf("%d", &faktor)

		fmt.Println(fak(faktor))
	}
}
